app/assets/javascripts/select-kit/addon/components/select-kit.js

```diff
@@ -65,6 +66,17 @@ export function pluginApiIdentifiers(identifiers) {
   };
 }
 
+export function resolveComponent(context, component) {
+  const owner = getOwner(context);
+
+  if (typeof component === "string") {
+    // TODO: derecate
```

Comment:
```suggestion
    // TODO: deprecate
```

deprecation to be added in another PR?

Reply:
Eventually we need to deprecate all these string lookups, but it needs some refactoring in core before we can do that. Will just remove the comment for now.